recipes/constyle/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "constyle" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ceb31c979b2f0ca279aa8b915085405926cf4f0325def271a0e0eb7a2c581096
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - poetry
+  run:
+    - python >=3.7
+    - importlib-metadata
```

```suggestion
    - importlib-metadata >=4.12.0,<5.0.0a
```

The logs failed with
```
+ pip check
constyle 0.2.0 has requirement importlib-metadata<5.0.0,>=4.12.0, but you have importlib-metadata 4.11.4.
Tests failed for constyle-0.2.0-pyhd8ed1ab_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
```

I think the trailing `a` helps against pre-release builds.